math.js

```diff
@@ -28,3 +28,11 @@ function testAdd() {
 function subtract(a, b) {
   return a - b;
 }
+
+function power(a, b) {
```

Comment:
Please add some js documentation for this function.

Reply:
resolved

Reply:
Can you please improve the names of the arguments. a and b are too generic.

Reply:
resolved